site/docs/guidelines/interaction-states.mdx

```diff
@@ -15,6 +15,20 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 ## To keep in mind
 
+### Focus indicator
+- Any element a user can interact with must have a focus indicator when tabbing while using keyboard navigation. This helps users with screen readers and limited mobility to understand what they are interacting with and where they are on the page. 
```

Comment:
This isn't really benefiting screen reader users (there's a good chance they won't be able to see a focus state!) but it helps with keyboard only users, voice control users, and other forms of assistive tech

```suggestion
- Any element a user can interact with must have a focus indicator when tabbing while using keyboard navigation. This helps users with assistive technology understand what they are interacting with and where they are on the page. 
```